### ISSUE TITLE
How to only use default values when the chart has a custom values file

### ISSUE THREAD

**Issue by ffdfgdfg** [vmware-tanzu/kubeapps#6564]
**Summary**
The chart has a custom values file, and dashboard only display custom values, but I want to use defaults.

**Background and rationale**

- rook-ceph-cluster added an external custom values file and released https://github.com/rook/rook/commit/3e61efc97611d03c3723607f245c3ab974fef52e

- kubeapps will merge custom values in this release
https://github.com/vmware-tanzu/kubeapps/releases/tag/v2.6.3

**Description**
I deployed rook-ceph-cluster with default values via kubeapps. After they added an external custom values file, upgrade form only display custom values, that's leads wrong values.
Deploy new rook-ceph-cluster chart also uses custom values and not usable.
**Additional context**
I tried rollback to v2.6.0, but when I click into upgrade, shows `An error occurred: Document with errors cannot be stringified.` , it's ok in v2.8.0

**Comment by absoludity** [comment]
Oh - interesting. Yes, we added a feature so that companies can add a custom values file to charts to have those values (merged with the defaults) presented to users, but hadn't anticipated public charts to add custom value files for their own uses.

Currently if there is more than one custom value file, we present a drop-down to choose from, but if there is only one, we do not provide any options.

Two options I can imagine working into the future:
- require a specifically named custom file for Kubeapps to assume it should be presented by default (such as `values-kubeapps-default.yaml`), or
- Always use a drop-down (but this isn't great for the original use-case.

Any objections to updating this functionality to require the specific filename for Kubeapps to present it as the default? I think that better suits the original use-case for that feature, and ensures we avoid this unexpected behaviour.

**Comment by absoludity** [comment]
Added the good first issue label as the fix, if agreed as above, should be straight forward and a good intro to the code-base for anyone interested.

**Comment by ffdfgdfg** [comment]
If the specifically named custom file not exists, will kubeapps use default values?

**Comment by absoludity** [comment]
> If the specifically named custom file not exists, will kubeapps use default values?

Yes, that's what I'm proposing: Kubeapps will always display the chart default `values.yaml` values unless
-  a file specifically named `values-kubeapps-default.yaml` is present, in which case it will be used instead, or
- multiple `values-([a-z1-9]+).yaml` files exist, in which case a drop-down selector will be displayed for those files (as is the case currently).

**Comment by ffdfgdfg** [comment]
> 

Excellent! Thank you.
